### ISSUE TITLE
Update Manager fails to update Nemo Actions listed

### ISSUE THREAD

**Issue by tonywhelan** [linuxmint/mint21.3-beta#20]
Update manager shows 3 Nemo Actions can be updated. But after pressing "Install updates", the actions re-appear in Update Manager.
This is irrespective of whether the actions have been installed or not.

![update screenshot](https://github.com/linuxmint/mint21.3-beta/assets/5572784/e9f22d2b-0ade-4903-bf91-67bfed328341)

**Comment by rcalixte** [comment]
Are all of the other system updates installed successfully? It may also be worth trying to uninstall and reinstall the actions to see if it was an artifact of the beta.

**Comment by clefebvre** [comment]
@rcalixte @mtwebster This seems to be fixed on master.

Here's what's happening in 6.0.0 when I tested it with send-with-kdeconnect@rcalixte:

- send-with-kdeconnect@rcalixte is installed
- I update it with mintupdate
 -> mintupdate says it's doing it..
 -> when the list of updates come back, the action is still listed as if nothing happened
- Looking in csettings actions, I see it's no longer installed.
- Looking in .local/share/nemo/actions...
 -> send-with-kdeconnect@rcalixte.nemo_action is gone
 -> send-with-kdeconnect@rcalixte/ is gone
 -> all its content (metadata, icon, po/*, .sh file..etc) was dumped in  .local/share/nemo/actions/
- I refresh mintupdate
  -> send-with-kdeconnect@rcalixte disappears from the list

So basically, the action is uninstalled instead of being updated. All its content is dumped in  .local/share/nemo/actions/. MintUpdate is confused until refreshed.

**Comment by clefebvre** [comment]
Running cinnamon master I can't reproduce the issue any longer. The action is properly updated, mintupdate restarts Cinnamon, the action disappears from the updates list in mintupdate after being updated.

**Comment by rcalixte** [comment]
Right. This was fixed in linuxmint/cinnamon#11965.

**Comment by clefebvre** [comment]
Fixed in cinnamon 6.0.1
